enclave/Enclave.edl

```diff
@@ -22,13 +22,17 @@ enclave {
     from "sgx_stdio.edl" import *;
     from "sgx_backtrace.edl" import *;
     from "sgx_tstdc.edl" import *;
+
     trusted
     {
         /* ECALLs */
         public sgx_status_t ecall_test([in, size=len] const uint8_t* some_string, size_t len);
```

Comment:
Can probably remove `ecall_test` function code as we're not using it in app/src/main.rs in another PR.